templates/resources.html

```diff
@@ -133,6 +133,39 @@ <h3 id="how-roblox">
       <h2>Must reads</h2>
     </div>
   </div>
+  
+  <div class="row">
+  <article class="p-strip is-shallow">
+    <div class="row">
+      <div class="col-4">
+        <h3 class="p-heading--4">
+          <a
+            href="https://ubuntu.com/engage/bare-metal-kubernetes"
+            >Easy and automated bare metal Kubernetes: The Definitive Whitepaper</a
+          >
+        </h3>
+      </div>
+      <div class="col-4">
+        <p>
+          Read the definitive whitepaper on bare metal Kubernetes featuring MAAS.
+        </p>
+        <p>
+          This whitepaper goes in depth into the history of VMs, how they compare with Kubernetes, and the important role bare metal orchestration (and MAAS) has to play.
+          Learn how to deploy Kubernetes on bare metal with MAAS, and the benefits that it has.
+          Designed to go hand in hand with the bare metal K8s video tutorial.
+        </p>
+      </div>
+      <div class="col-4 u-align--center">
+        <img
+          src="https://assets.ubuntu.com/v1/196ebe18-MAAS+Kubernetes.svg"
+          width="200"
+          height="200"
+        />
```

Comment:
Could you please add an alt tag? I'd suggest `alt=""` as this image seems purely decorative. 

More info here: https://www.w3.org/WAI/tutorials/images/decorative/